tests/shell/pipeline/commands/internal.rs

```diff
@@ -989,7 +989,7 @@ fn hide_alias_hides_alias() {
         "
     ));
 
-    assert!(actual.err.contains("did you mean 'all'?"));
+    assert!(actual.err.contains("Command `ll` not found"));
```

Comment:
I think it's good to keep `did you mean` error message, we should assert error message contains `Did you mean 'all'`, if that is possible.

Reply:
I'll do that. Though I do wonder if these tests are a bit fragile.